### DISCUSSION TITLE
Project sync to QFieldCloud failure

### DISCUSSION THREAD

**Discussion by NJHall94** [opengisch/QField#3822] [Q&A]
I'm very new to this and wonder if anyone can help. When trying to sync my QGIS project to Qfield cloud, the project file fails to upload with the error: [Multiple projects] multiple projects. All other layers and files upload without issue. I'm happy to provide any other information needed.

Thanks for any advice in advance,
Nick

**Comment by suricactus** [comment]
My initial guess is that you have multiple QGIS (.qgs/.qgz) files in the same directory OR you have renamed you QGIS project file locally and you try to reupload it again.

The easiest solution would be to either rename the file to it's old name, or delete the server stored QGIS project file.

BTW, providing username, project name and .zip of your project greatly helps to support you.

**Reply by NJHall94** [reply]
Good evening, 

Many thanks for the advice on support and for providing a solution! I didn't have multiple files in the directory, but deleting the server stored version and re-uploading fixed the issue. 

Thanks again!
Nick
